modules/appeals_api/app/services/appeals_api/evidence_submission/file_validator.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+require 'pdf_info'
+
+module AppealsApi
+  module EvidenceSubmission
+    class FileValidator
+      include SentryLogging
+
+      class UploadValidationError < StandardError; end
+
+      MAX_PAGE_SIZE = { width: 11, height: 11 }.freeze
+      MAX_FILE_SIZE = 100.megabytes
+
+      def initialize(file)
+        @file = file
+        @filename = File.basename(@file)
+      end
+
+      def call
+        return file_type_error unless pdf_metadata_present?
+        return file_size_error unless valid_file_size?
+        return max_dimension_error unless valid_page_dimensions?
+
+        validated
+      end
+
+      private
+
+      def pdf_metadata_present?
+        @pdf_metadata = PdfInfo::Metadata.read(@file)
+      rescue PdfInfo::MetadataReadError => e
+        log_exception_to_sentry(e, {}, {}, :warn)
+      ensure
+        @pdf_metadata.present?
+      end
+
+      def valid_file_size?
+        current_file_size <= MAX_FILE_SIZE
+      end
+
+      def current_file_size
+        @file_size ||= @file.size
+      end
+
+      def file_size_mb
+        current_file_size / 1_000_000
+      end
+
+      def valid_page_dimensions?
+        @dimensions = @pdf_metadata.page_size_inches
+        @dimensions[:height] <= MAX_PAGE_SIZE[:height] && @dimensions[:width] <= MAX_PAGE_SIZE[:width]
+      end
+
+      def file_type_error
+        msg = I18n.t('appeals_api.evidence_submission.pdf_read_error')
+        log_exception_to_sentry(UploadValidationError.new(msg), {}, {}, :warn)
+        {
+          title: 'invalid file type',
```

Comment:
`exceptions.en.yml`appears to have the first letter of the title capitalized - let's follow the same format.